products/cloudbuild/api.yaml

```diff
@@ -121,19 +121,19 @@ objects:
           - !ruby/object:Api::Type::String
             name: 'branchName'
             description: |
-              Name of the branch to build.
+              Name of the branch to build. A branch name, tag, or commit SHA must be provided.
           - !ruby/object:Api::Type::String
             name: 'tagName'
             description: |
-              Name of the tag to build.
+              Name of the tag to build. A branch name, tag, or commit SHA must be provided.
```

Comment:
On a quick read it appeared that all 3 of these values could be passed in to this field. Maybe rephrase to "One of these must be provided..."?